src/traces/bar/cross_trace_calc.js

```diff
@@ -713,6 +712,21 @@ function normalizeBars(sa, sieve, opts) {
     }
 }
 
+// Add an `_sMin` and `_sMax` value for each bar representing the min and max size value
+// across all bars sharing the same position as that bar. These values are used for rounded
+// bar corners, to carry rounding down to lower bars in the stack as needed.
+function setHelperValuesForRoundedCorners(calcTraces, sMinByPos, sMaxByPos) {
+    // Set `_sMin` and `_sMax` value for each bar
```

Comment:
We only need these when having bars with rounded corners.
This function is pretty slow and it should not be called when it is not really needed.

Reply:
@archmoj Yes good point. Would it make sense then to check whether any of the traces in `calcTraces` have a `cornerradius` value, and skip this function if not?

Reply:
Yes that's a good idea.